build.gradle

```diff
@@ -9,311 +9,401 @@ buildscript {
+    implementation 'com.zaxxer:HikariCP:5.0.1'
+    implementation 'org.jocl:jocl:2.0.1'
+
+    implementation 'org.jooq:jooq:3.20.3'
+    implementation 'org.flywaydb:flyway-core:8.5.13'
+    implementation 'org.flywaydb:flyway-mysql:8.5.13'
+    implementation 'org.postgresql:postgresql:42.7.3'
+    implementation 'com.kohlschutter.junixsocket:junixsocket-core:2.8.3'
+    implementation 'org.mariadb.jdbc:mariadb-java-client:2.4.1'
+    implementation 'org.xerial:sqlite-jdbc:3.45.2.0'
+
+    implementation 'org.owasp.encoder:encoder:1.2.3'
+    implementation 'org.bitlet:weupnp:0.1.4'
+    implementation 'com.google.zxing:core:3.4.1'
+    implementation 'com.google.zxing:javase:3.4.1'
+    implementation 'org.slf4j:slf4j-api:1.7.35'
+    implementation 'org.slf4j:slf4j-jdk14:1.7.35'
+
+    implementation 'org.eclipse.jetty:jetty-server:10.0.24'
+    implementation 'org.eclipse.jetty:jetty-servlet:10.0.24'
+    implementation 'org.eclipse.jetty:jetty-servlets:10.0.24'
+    implementation 'org.eclipse.jetty:jetty-rewrite:10.0.24'
+    implementation 'org.eclipse.jetty.websocket:websocket-jetty-server:10.0.24'
+
+    implementation 'javax.annotation:javax.annotation-api:1.3.2'
+    implementation 'com.github.jiconfont:jiconfont:1.0.0'
+    implementation 'com.github.jiconfont:jiconfont-swing:1.0.1'
+    implementation 'com.github.jiconfont:jiconfont-font_awesome:4.7.0.1'
+    implementation 'org.apache.logging.log4j:log4j-api:2.17.1'
+    implementation 'org.apache.logging.log4j:log4j-core:2.17.1'
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.15.2'
+    implementation 'org.eclipse.jgit:org.eclipse.jgit:6.5.0.202303070854-r'
+    implementation 'jakarta.xml.bind:jakarta.xml.bind-api:3.0.1'
+
+
+    testImplementation("org.junit.platform:junit-platform-engine:1.5.1")
+    testImplementation("org.junit.platform:junit-platform-launcher:1.5.1")
+    testImplementation("org.junit.jupiter:junit-jupiter-api:5.5.1")
+    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.5.1")
+    testRuntimeOnly("org.junit.vintage:junit-vintage-engine:5.5.1")
+
+    testImplementation 'org.apache.httpcomponents:httpclient:4.5.13'
+    testImplementation 'org.mockito:mockito-core:5.11.0'
+    testImplementation 'org.powermock:powermock-module-junit4:2.0.9'
+    testImplementation 'org.powermock:powermock-api-mockito2:2.0.9'
 }
 
-
 test {
-  useJUnitPlatform()
-  testLogging {
-    exceptionFormat "full"
-  }
+    useJUnitPlatform()
+    testLogging {
+        exceptionFormat "full"
+        showStandardStreams = true
+        events "passed", "skipped", "failed"
+    }
+
+    jvmArgs += [
+        "--add-opens=java.base/java.lang=ALL-UNNAMED",
+        "--add-opens=java.base/java.util=ALL-UNNAMED",
+        "--add-opens=java.base/java.io=ALL-UNNAMED",
+        "--add-opens=java.base/java.net=ALL-UNNAMED",
+        "--add-opens=java.base/jdk.internal.reflect=ALL-UNNAMED"
+    ]
+
+    exclude '**/ATTest.class',
```

Comment:
hmmm...not ideal

Reply:
yes, I think this work needs to be done as soon as we have it on testnet ... was not able to build without excluding them